addons/base_import/static/src/xml/base_import.xml

```diff
@@ -173,55 +118,196 @@
-                </ul>
-                <t t-if="error_value[0].moreinfo" t-raw="info(error_value[0].moreinfo)"/>
 
-            </t>
-            <t t-else="">
-                <t t-call="ImportView.error.single">
-                    <t t-set="error" t-value="error_value[0]"/>
+    <t t-name="ImportView.error">
+        <div t-foreach="errors.length" t-as="index" t-attf-class="oe_import_report alert alert-#{errors[index].type}">
+            <span class="oe_import_report_message">
+                <b><t t-esc="errors[index].message"/></b>
+                <t t-if="errors[index].rows" t-call="ImportView.error.at">
+                    <t t-set="rows" t-value="errors[index].rows"/>
                 </t>
+            </span>
+            <t t-if="errors[index].moreinfo" t-call="ImportView.error.more_info">
+                <t t-set="messages" t-value="errors[index].moreinfo"/>
             </t>
-        </li>
-    </ul>
-    <t t-name="ImportView.error.multi.header">
-        <span class="oe_import_report_message">
-            <t t-esc="error.message"/>
-            <t t-esc="at_multi_header(error_length)"/>
-        </span>
-    </t>
-    <t t-name="ImportView.error.multi.body">
-        <span class="oe_import_report_message">
-            <t t-esc="at_multi(error.rows)"/>
-        </span>
+        </div>
     </t>
+
+    <div t-name="ImportView.fieldError" t-attf-class="oe_import_report alert alert-#{errors[0].type}">
+        <t t-if="errors.length gt 1" t-call="ImportView.error.multi"></t>
+        <t t-else="" t-call="ImportView.error.single">
+            <t t-set="error" t-value="errors[0]"/>
+        </t>
+    </div>
     <t t-name="ImportView.error.single">
         <span class="oe_import_report_message">
-            <t t-esc="error.message"/>
+            <t t-if="error.value &amp;&amp; !error.error_message">
+                No matching records found for <t t-esc="error.field_type"/> <b>'<t t-esc="error.value"/>'</b> in field <b t-esc="error.field_name"/>
+            </t>
+            <b t-else="" t-esc="error.message"/>
         </span>
-        <t t-if="error.rows"  t-esc="at(error.rows)"/>
-        <t t-if="error.moreinfo" t-raw="info(error.moreinfo)"/>
+        <t t-if="error.rows" t-call="ImportView.error.at">
+            <t t-set="rows" t-value="error.rows"/>
+        </t>
+        <t t-if="error.moreinfo" t-call="ImportView.error.more_info">
+            <t t-set="messages" t-value="error.moreinfo"/>
+        </t>
+    </t>
+
+    <t t-name="ImportView.error.multi">
+        <div class="oe_import_report_message">
+            <t t-if="errors[0].value">
+            No matching records found for the following <t t-esc="errors[0].field_type"/> </t>
+            <t t-else="">Multiple errors occured </t>
+            in field <b><t t-esc="errors[0].field_name"/></b>:
+            <ul>
+                <t t-foreach="errors.length" t-as="index">
+                    <li t-att-class="index gt 2 ? 'oe_import_report_more':''">
```

Comment:
`"index gt 2 ? 'oe_import_report_more':''"`
I don'y see how this can work Oo

Reply:
There's qweb magic because you can't put `<`, `&` or `>` in attributes without escaping them, so qweb has psedo operators `and`, `or`, `lt[e]` and `gt[e]`: https://github.com/odoo/odoo/blob/7b763176ba76dbfdc24d8c646fedd5355b4f62da/addons/web/static/lib/qweb/qweb.js#L77-L82

Reply:
The more you know ^^
Thanks for the explanation.